skedge-backend/middlewares/decrypt-requests.js

```diff
@@ -0,0 +1,21 @@
+const decrypt = require("../web_api_utilities/rsa-encryption").decryptStringWithRsaPrivateKey
+
+module.exports = function (req, res, next) {
+    try {
+        if(req.method == "POST" && req.body != undefined)
+        {
+            console.log("---------------------------------------------------------\nReceived encrypted data:");
```

Comment:
These look like testing logs that we might want to remove before merging to master

Reply:
These are useful [info] logs for the team and don't take up much room in the console.
I would suggest we keep them.